CrowdStrike/crowdstrike_falcon/ingest/parser.yml

```diff
@@ -0,0 +1,138 @@
+name: 'crowdstrike falcon edr'
+pipeline:
+  - name: stage1
+    description: ''
+    filter: ''
+    external:
+      name: json.parse-json
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: read_event_date
+    description: ''
+    filter: ''
+    external:
+      name: date.parse
+      properties:
+        input_field: '{{stage1.message.metadata.eventCreationTime/1000}}'
+        output_field: test
+        format: timestamp
+        timezone: UTC
+  - name: set_event_date
+    description: ''
+    filter: ''
+    external: null
+  - name: streamstarted
+    description: ''
+    filter: '{{stage1.message.event.OperationName == "streamStarted"}}'
+    external: null
+  - name: streamstopped
+    description: ''
+    filter: '{{stage1.message.event.OperationName == "streamStopped"}}'
+    external: null
+  - name: detectionsummaryevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "DetectionSummaryEvent"}}'
+    external: null
+  - name: useractivityauditevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "UserActivityAuditEvent"}}'
+    external: null
+  - name: authactivityauditevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "AuthActivityAuditEvent"}}'
+    external: null
+  - name: incidentsummaryevent
+    description: ''
+    filter: '{{stage1.message.metadata.eventType == "IncidentSummaryEvent"}}'
+    external: null
+stages:
+  set_event_date:
+    actions:
+      - set:
+          '@timestamp': '{{read_event_date.test}}'
+          crowdstrike.event_type: '{{stage1.message.metadata.eventType}}'
+        filter: ''
+        name: set
+  streamstarted:
+    actions:
+      - set:
+          source.ip: '{{stage1.message.event.UserIp}}'
+          event.kind: event
+          event.type: start
+          service.name: '{{stage1.message.event.ServiceName}}'
+          event.category: session
+          source.user.id: '{{stage1.message.event.UserId}}'
+        name: set
+  streamstopped:
+    actions:
+      - set:
+          source.ip: '{{stage1.message.event.UserIp}}'
+          event.kind: event
+          event.type: stop
+          service.name: '{{stage1.message.event.ServiceName}}'
+          event.category: session
+          source.user.id: '{{stage1.message.event.UserId}}'
+        name: set
+  detectionsummaryevent:
+    actions:
+      - set:
+          host.ip: '{{stage1.message.event.LocalIP}}'
+          agent.id: '{{stage1.message.event.AgentIdString}}'
+          host.mac: '{{stage1.message.event.MACAddress}}'
+          file.name: '{{stage1.message.event.FileName}}'
+          file.path: '{{stage1.message.event.FilePath}}'
+          host.name: '{{stage1.message.event.ComputerName}}'
```

Comment:
Possible to add log.hostname as well for temporary compatibility with old ECS schema (used in our current rule catalogue and old parsers)?